website/docs/docs/build/incremental-microbatch.md

```diff
@@ -179,12 +179,14 @@ It does not matter whether the table already contains data for that day. Given t
 
 Several configurations are relevant to microbatch models, and some are required:
 
+
 | Config   |  Description   | Default | Type | Required  |
 |----------|---------------|---------|------|---------|
 | [`event_time`](/reference/resource-configs/event-time)  | The column indicating "at what time did the row occur." Required for your microbatch model and any direct parents that should be filtered.   | N/A     |  Column  |  Required |
 | `begin`      |  The "beginning of time" for the microbatch model. This is the starting point for any initial or full-refresh builds. For example, a daily-grain microbatch model run on `2024-10-01` with `begin = '2023-10-01` will process 366 batches (it's a leap year!) plus the batch for "today."        | N/A     | Date   | Required |
 | `batch_size` |  The granularity of your batches. Supported values are `hour`, `day`, `month`, and `year`    | N/A     | String  | Required |
 | `lookback`   | Process X batches prior to the latest bookmark to capture late-arriving records.    | `1`     | Integer | Optional |
+|`concurrent_batches`| Configures whether batches can run concurrently (at the same time) or sequentially (one after the other). Can set to `True` or `False`. When set to `True` and conditions are met, batches run in parallel. | `None` | Boolean | Optional |
```

Comment:
I worry that people reading the first sentence will think they _have_ to use `concurrent_batches` to get batches to run in parallel. Would changing the sentence to something like 
> An override for whether batches run concurrently (at the same time) or sequentially (one after the other).

make sense?

Reply:
Hiya @QMalcolm 

This makes sense. I'll change the sentence to your suggestion.

Kind Regards
Natalie